app/connectors/IncomeTaxViewChangeConnector.scala

```diff
@@ -63,7 +63,7 @@ trait IncomeTaxViewChangeConnector extends RawResponseReads with FeatureSwitchin
     s"${appConfig.itvcProtectedService}/income-tax-view-change/nino-lookup/$mtdRef"
   }
 
-  def getReportDeadlinesUrl(nino: String): String = {
+  def getNextUpdatesUrl(nino: String): String = {
```

Comment:
Just a note: with this change the url and the name of the function go out of sync. Url says `report-deadlines`, the function says `NextUpdates`

Reply:
Changed back to "getReportDeadlinesUrl"